docs/reference/contrib/forms/customisation.md

```diff
@@ -1,17 +1,17 @@
-Form builder customisation
-==========================
+# Form builder customisation
 
+```eval_rst
 For a basic usage example see :ref:`form_builder_usage`.
+```
 
-Custom ``related_name`` for form fields
----------------------------------------
+## Custom `related_name` for form fields
 
-If you want to change ``related_name`` for form fields
-(by default ``AbstractForm`` and ``AbstractEmailForm`` expect ``form_fields`` to be defined),
-you will need to override the ``get_form_fields`` method.
+If you want to change `related_name` for form fields
+(by default `AbstractForm` and `AbstractEmailForm` expect ``form_fields`` to be defined),
+you will need to override the `get_form_fields` method.
 You can do this as shown below.
 
-.. code-block:: python
+```python
```

Comment:
Switching to Markdown fenced code blocks, we need to remove the indentation and the first empty newline. Otherwise both show up in the output.